sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/namedExpressions.scala

```diff
@@ -71,11 +71,11 @@ trait NamedExpression extends Expression {
   def exprId: ExprId
 
   /**
-   * Returns a dot separated fully qualified name for this attribute.  Given that there can be
-   * multiple qualifiers, it is possible that there are other possible way to refer to this
-   * attribute.
+   * Returns a dot separated fully qualified name for this attribute.  If the name or any qualifier
+   * contains `dots`, it is quoted to avoid confusion.  Given that there can be multiple qualifiers,
```

Comment:
What does "`dots`" mean here - what does it refer to?

Reply:
I stuck to existing wording for consistency:
https://github.com/apache/spark/blob/116f6a719eaf1220d8eed98433f724fca672acc6/sql/catalyst/src/main/scala/org/apache/spark/sql/sources/filters.scala#L83-L85
https://github.com/apache/spark/blob/116f6a719eaf1220d8eed98433f724fca672acc6/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFiltersBase.scala#L45-L48
https://github.com/apache/spark/blob/116f6a719eaf1220d8eed98433f724fca672acc6/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala#L58-L60

Reply:
OK. That's weird. I wonder if the comment means to say `.`